src/services/rooms.service.js

```diff
@@ -0,0 +1,24 @@
+const { Room } = require('../models/room.model');
+
+const create = async (room) => {
+  return Room.create(room);
```

Comment:
Consider adding validation for the `room` object to ensure it meets the expected format and constraints before attempting to create a new room.